src/hlstrafe.cpp

```diff
@@ -1998,6 +2011,9 @@ namespace HLStrafe
 		out.NextFrameIs0ms = false;
 
 		float yaw = static_cast<float>(NormalizeDeg(out.Yaw));
+
+		curState.LastYaw = yaw;
```

Comment:
I think you forgot to commit hlstrafe.hpp changes

Reply:
Also it's a little weird to use a curState variable for something that seems to be fully local to MainFunc but it's understandable